schema/deploy/warehouse/site/data.sql

```diff
@@ -178,11 +178,23 @@ insert into warehouse.site(identifier, details)
         ('UWReopeningSwabNSend',                    '{"category": "community",  "type": "uw-reopening"}'),
         ('UWReopeningDropbox',                      '{"category": "community",  "type": "uw-reopening",
             "swab_site": "uw_box"}'),
+        ('UWReopeningDropboxTacoma',                '{"category": "community",  "type": "uw-reopening",
```

Comment:
I wonder whether these should have `type: collegeCampus` or `type: uw-reopening` - its a bit of a can of worms, maybe best discussed at standup tomorrow.

Reply:
Thanks for the brief discussion at standup this morning about this. You think we should talk more about it along with these changes? I can add a Trello card for it if not so we don't forget to revisit.

Reply:
I think let's go ahead with the database portion of these changes (as-is is fine for now!!) and just drop the etl changes as you did already!